storage/remote/write_handler_test.go

```diff
@@ -852,6 +852,19 @@ func requireEqual(t *testing.T, expected, actual interface{}, msgAndArgs ...inte
 		msgAndArgs...)
 }
 
+func requireContainsSample(t *testing.T, actual []mockSample, expected mockSample) {
```

Comment:
Can you remove this to storage/remote/write_test.go? It doesn't make sense in a test file where it isn't used. It only makes reading/reviewing trickier.